vendor/assets/stylesheets/dvl/core/forms.scss

```diff
@@ -383,7 +383,7 @@ form {
   }
 }
 
-$styledSelectHeight: 2.25rem;
+$styledSelectHeight: 2.5rem;
```

Comment:
Yes, `14*2.25` (our root font size * this value) is `31.5px`. I was getting +/-1px vertical alignment errors because of this. `14*2.5` is `35px`, which solves the problem.